strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-npm-layout-provider/src/test/java/org/carlspring/strongbox/artifact/generator/NpmPackageGenerator.java

```diff
@@ -41,6 +42,9 @@
 
     private ObjectMapper mapper = new ObjectMapper();
 
+    public NpmPackageGenerator()
```

Comment:
@yugander-krishan-singh we don't need noarg constructor here.

Reply:
Sure. Removed it. I was thinking it might be required for writing the unit-tests.